Sources/Plot/API/HTMLElements.swift

```diff
@@ -319,6 +325,12 @@ public extension Node where Context: HTML.BodyContext {
         .element(named: "select", nodes: nodes)
     }
 
+    /// Add a `<source>` HTML element within the current context.
+    /// - parameter nodes: The element's attributes and child elements.
+    static func source(_ attributes: Attribute<HTML.SourceSetContext>...) -> Node {
```

Comment:
This doesn't seem correctly scoped, since `source` element's can't appear anywhere within an HTML body, right? Probably needs another context.